misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/README.md

```diff
@@ -1,23 +1,30 @@
 ## Current data mapping
+
 ****
-Values computed in Metasfresh 
+Values computed in Metasfresh
+
 * `Order` - pulled via the search endpoint `api/v3/search/order/`
 
 computed from the following parameters set on window `541116 - ExternalSystem_Config_Shopware6`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerID`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerLocationID`
-  * `JsonExternalSystemRequest.parameters.JSONPathConstantSalesRepID`
-  * `JsonExternalSystemRequest.parameters.ClientId`  
-  * `JsonExternalSystemRequest.parameters.ClientSecret`  
-  * `JsonExternalSystemRequest.parameters.BasePath`
-  * `JsonExternalSystemRequest.parameters.UpdatedAfterOverride` 
-  * `JsonExternalSystemRequest.parameters.UpdatedAfter`
-    
+
+* `JsonExternalSystemRequest.parameters.ClientId`
+* `JsonExternalSystemRequest.parameters.ClientSecret`
+* `JsonExternalSystemRequest.parameters.BasePath`
+* `JsonExternalSystemRequest.parameters.JSONPathConstantBPartnerLocationID`
+* `JsonExternalSystemRequest.parameters.JSONPathEmail`
+* `JsonExternalSystemRequest.parameters.JSONPathConstantSalesRepID`
+* `JsonExternalSystemRequest.parameters.UpdatedAfterOverride`
+* `JsonExternalSystemRequest.parameters.UpdatedAfter`
+* `JsonExternalSystemRequest.parameters.M_FreightCost_NormalVAT_Product_ID`
+* `JsonExternalSystemRequest.parameters.FreightCost_NormalVAT_Rates`
+* `JsonExternalSystemRequest.parameters.M_FreightCost_ReducedVAT_Product_ID`
+* `JsonExternalSystemRequest.parameters.FreightCost_Reduced_VAT_Rates`
+
 * `JsonExternalSystemShopware6ConfigMappings` - computed from mappings set on window `541116 - ExternalSystem_Config_Shopware6` send on `JsonExternalSystemRequest.parameters.ConfigMappings`
 
 **Shopware6 => metasfresh BPartners**
```

Comment:
please iprove the whole document by using section headers (`#`, `##`, ...) instead of bold